doc/SONiC_202106_Release_Notes.md

```diff
@@ -0,0 +1,180 @@
+# SONiC 202106 Release Notes
+
+This document captures the new features added and enhancements done on existing features/sub-features for the SONiC [202106](https://github.com/Azure/SONiC/wiki/Release-Progress-Tracking-202106) release.
+
+
+
+# Table of Contents
+
+ * [Branch and Image Location](#branch-and-image-location)
+ * [Dependency Version](#dependency-version)
+ * [Security Updates](#security-updates)
+ * [Feature List](#feature-list)
+ * [SAI APIs](#sai-apis)
+ * [Contributors](#contributors)
+
+
+# Branch and Image Location  
+
+Branch : https://github.com/Azure/sonic-buildimage/tree/202106 <br>
+Image  : https://sonic-jenkins.westus2.cloudapp.azure.com/  (Example - Image for Broadcom based platforms is [here]( https://sonic-jenkins.westus2.cloudapp.azure.com/job/broadcom/job/buildimage-brcm-202106/lastSuccessfulBuild/artifact/target/))
+
+# Dependency Version
+
+|Feature                    | Version  |
+| ------------------------- | --------------- |
+| Linux kernel version      | linux_4.19.0-12-2 (4.19.152-1)   |
+| SAI   version             | SAI v1.8.1    |
+| FRR                       | 7.5.1   |
+| LLDPD                     | 1.0.4-1    |
+| TeamD                     | 1.28-1    |
+| SNMPD                     | 5.7.3+dfsg-1.5    |
+| Python                    | 3.6.0-1    |
+| syncd                     | 1.0.0    |
+| swss                      | 1.0.0    |
+| radvd                     | 2.17-2~bpo9+1    |
+| isc-dhcp                  |  4.4.1-2   |
+| sonic-telemetry           | 0.1    |
+| redis-server/ redis-tools | 5.0.3-3~bpo9+2    |
+
+
+# Security Updates
+
+1. Kernel upgraded from 4.9.168-1+deb9u5 (SONiC Release 202006) to 4.19.152-1 for SONiC release.
+   Change log: https://salsa.debian.org/kernel-team/linux/-/raw/buster-security/debian/changelog 
+
+2. Plexus-utils fix: CVE-2017-1000487   
+
+3. Docker upgraded from 18.09.2\~3-0\~debian-stretch to 18.09.8\~3-0\~debian-stretch. 
```

Comment:
docker is upgraded to buster (20.10.8)

Reply:
@tonytitus Thanks. Can you please share the reference link for this upgrade

Reply:
The same is addressed in Security section